src/SUMMARY.md

```diff
@@ -19,11 +19,16 @@
   * [Lists](concepts/html/lists.md)
   * [Elements](concepts/html/elements.md)
   * [Literals & Expressions](concepts/html/literals-and-expressions.md)
-  * [Components](concepts/html/components.md)
 * [Components](concepts/components/README.md)
+  * [Internal State](concepts/components/internalstate.md)
+  * [Trap events](concepts/components/trapevents.md)
   * [Properties](concepts/components/properties.md)
-  * [Callbacks](concepts/components/callbacks.md)
+  * [Emit events](concepts/components/emitevents.md)
+  * [Nested component](concepts/components/nestedcomponents.md)
   * [Refs](concepts/components/refs.md)
+  * [Styling](concepts/components/styling.md)
+  * [Life cycle](concepts/components/lifecycle.md)
```

Comment:
Stylistically, I think it's better for "life cycle" to be "lifecycle."

Reply:
I ask myself about it and google it. Found more about "Life Cycle"
Anyway ... we are 2 having the same feeling => I change to 1 word